pages/[id].jsx

```diff
@@ -1,8 +1,12 @@
 import Head from 'next/head';
 import SkipToContent from '../components/SkipToContent';
+import TradingCardImage from '../components/TradingCardImage';
 import apiService from '../services/apiService';
 
 export default function DetailPage({ fetchedTeamMember }) {
+  // Temporary random number generator to determine team member hours
+  const randomHours = () => Math.floor(Math.random() * 33);
```

Comment:
As mentioned, feel free to add `hoursForecasted` to the mock data instead of using random numbers.